css/style.css

```diff
@@ -1,9 +1,276 @@
+* {
+  margin: 0%;
+  box-sizing: border-box;
+}
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+a {
+  text-decoration: none;
+  line-height: 1.5;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+.top-bar {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding: 16px 90px 16px 90px;
+}
+
+.karma-logo {
+  padding: 10px;
+}
+
+.karma {
+  width: 40px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+nav ul li {
+  display: inline;
+  padding: 10px;
+  font-size: 20px;
+}
+
+.navbar-links {
+  font-size: 25px;
+  color: rgb(169, 135, 135);
+}
+
+nav ul li a:hover {
+  color: rgb(255, 72, 0);
+}
+
+.meet-karma {
+  color: black;
+}
+
+nav ul li a:active {
+  color: #000;
+}
+
+.header-mobile {
+  visibility: hidden;
+}
+
+.first-part {
+  background-image: url(/home/trip/Documents/HTML-CSS-Coursework-Week2/img/first-background.jpg);
+  background-size: cover;
+  background-position: center center;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+h1,
+h2 {
+  color: white;
+}
+
+h1 {
+  font-size: 70px;
+  font-weight: 100;
+  margin-bottom: 10px;
+}
+
+h2 {
+  font-size: 50px;
+  font-weight: 100;
+  margin-top: 30px;
+}
+
+.learn-button {
+  margin-top: 50px;
+}
+
+.orange-button {
+  background-color: rgb(255, 72, 0);
+  border: none;
+  color: white;
+  border-radius: 4px;
+  padding: 25px 35px 25px 35px;
+  font-size: 22px;
+}
+
+.second-part {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+h3 {
+  text-align: center;
+  font-size: 70px;
+  padding: 100px;
+  font-weight: 100;
+}
+
+h4 {
+  font-size: 40px;
+  font-weight: 100;
+  padding: 50px;
+}
```

Comment:
I will change the font size and weight, the reason I didn't do `text-align` is because I did on the parent element `.first-part: flex-direction: column;
 justify-content: center;
 align-items: center;`